.circleci/check_gotest.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+# run test in each go environment and create $FILEPATH_ERROR_FLAG file if was any error. But all tests should
```

Comment:
I guess we should finish the comme— wait, someone's knocking at the door, I'll go take a look.

```suggestion
# Run test in each go environment and log errors to $FILEPATH_ERROR_FLAG.
# If all tests pass successfully then the file will not be created at all.
```